docs/conops.md

```diff
@@ -1,23 +1,23 @@
-![Arrow Banner](https://github.com/Arrow-air/tf-github/raw/main/src/templates/doc-banner-services.png)
+![Aetheric Banner](https://github.com/Arrow-air/tf-github/raw/main/src/templates/doc-banner-services.png)
```

Comment:
should point to https://github.com/aetheric-oss/.github/blob/main/assets/doc-banner.png